apps/codeforafrica/src/components/NavBarNavList/index.js

```diff
@@ -0,0 +1,72 @@
+import { Link } from "@commons-ui/next";
+import Box from "@mui/material/Box";
+import { useTheme } from "@mui/material/styles";
+import SvgIcon from "@mui/material/SvgIcon";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { ReactComponent as TwitterDesktopIcon } from "@/codeforafrica/assets/twitterDesktop.svg";
+import { ReactComponent as TwitterMobileIcon } from "@/codeforafrica/assets/twitterMobile.svg";
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function NavBarNavList({ menu, direction }) {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+
+  if (!menu?.length) {
+    return null;
+  }
+  return (
+    <Box component="nav">
+      <NavList direction={direction} sx={{ alignItems: "flex-start" }}>
+        {menu.map((item) => (
+          <NavListItem key={item.label} sx={{ m: "20px" }}>
+            <Link
+              href={item.href}
+              color="inherit"
+              underline="none"
+              variant={isMobile ? "h4" : "subtitle1"}
+              fontWeight={isMobile ? 700 : 400}
+              sx={{
+                "&:hover, &:active, &:focus, &:focus-within": {
+                  textDecoration: "none",
+                  color: { xs: "inherit", md: "primary.main" },
+                },
+              }}
```

Comment:
```suggestion
              sx={{
                typography: {{ xs: "h4", md: "body3" }}
                "&:hover, &:active, &:focus, &:focus-within": {
                  textDecoration: "none",
                  color: { xs: "inherit", md: "primary.main" },
                },
              }}
```
Avoid using `isMobile` since it's computed client-side.